sms-checker/notification_lambda.py

```diff
@@ -7,7 +7,13 @@ def lambda_handler(event,context) -> None:
     '''Runs the notification pipeline'''
     logger = logging.getLogger(__name__)
     logging.basicConfig(level=logging.INFO)
-
+    subscribers = get_subscribers()
+    email_list = [entry for entry in subscribers if entry['email']]
+    phone_list = [entry for entry in subscribers if entry['phone']]
+    if email_list:
+        send_email(email_list)
+    if phone_list:
+        send_sms(phone_list)
 
 if __name__ == '__main__':
```

Comment:
Works for testing purposes, but Dan made a point last week of removing all things that are unnecessary or 'extra' to our final functionality. I wonder if this falls into that category.